Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean

```diff
@@ -417,6 +417,7 @@ theorem adjunction_unit_app {X : Scheme} :
 -- Porting Note: Commented
 -- attribute [local semireducible] locallyRingedSpaceAdjunction ΓSpec.adjunction
 
+set_option maxHeartbeats 400000 in
```

Comment:
any idea what changed here / how to make this better / avoid the `convert`?

Reply:
The proof uses 200959 heartbeats and basically the same on master so I suspect this is just noise